release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -521,6 +521,68 @@ If the following conditions are true, the feature is enabled by default:
+If enabled, Metrics Server, rather than Prometheus Adapter, collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs.
+Using Metrics Server instead of Prometheus Adapter frees the core platform Prometheus stack from handling this functionality.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#metricsserverconfig[MetricsServerConfig] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-new-feature-to-send-exemplar-data-to-remote-write-storage-for-user-defined-projects"]
+==== New feature to send exemplar data to remote write storage for user-defined projects
+With this release, your user-defined projects can now use remote write to send exemplar data scraped by Prometheus to remote storage.
+To use this feature, configure remote write using the `sendExemplars` option in the `RemoteWriteSpec` resource.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#remotewritespec[RemoteWriteSpec] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-impoved-alert-querying-for-user-defined-projects"]
+==== Improved alert querying for user-defined projects
+With this release, applications in your user-defined projects now have API access to query alerts for application namespaces via the rules tenancy port for Thanos Querier. 
+Specifically, you can now constuct queries that access the `/api/v1/alerts` endpoint via port 9093 for Thanos Querier, provided that the HTTP request contains a `namespace` parameter.
+In previous releases, the rules tenancy port for Thanos Querier did not provide API access to the `/api/v1/alerts` endpoint.
+
+[id="ocp-4-15-monitoring-prometheus-updated-to-tolerate-scrape-time-jitters"]
+==== Prometheus updated to tolerate scrape time jitters to reduce TSDB storage requirements
+With this release, the default Prometheus configuration in the in-cluster monitoring stack has been changed so that jitters are tolerated at scrape time, which improves chunk compression for data storage.
+As a result, the time series database uses significantly less disk space.
+Note that this change can reduce the accuracy of derived data, such as the running average of a time series.
+
+[id="ocp-4-15-monitoring-improved-staleness-handling-for-the-kubelet-service-monitor"]
+==== Improved staleness handling for the kubelet service monitor
+With this release, staleness handling for the kubelet service monitor has been improved in order to ensure that alerts and time aggregations are accurate.
+This improved functionality is active by default and makes the dedicated service monitors feature obsolete.
+As a result, the dedicated service monitors feature is now deprecated and will be removed in a future {product-title} release.
```

Comment:
```suggestion
As a result, the dedicated service monitors feature is now deprecated and will be removed in a future release. In this version, the setting has no effect.
```

Reply:
Edited to be more specific that the setting now has no effect in this release